README.md

```diff
@@ -338,24 +338,32 @@ const count = 2 // it tries to run 2(), but 2 is not a function
 
 ## ![✔] 3.6 Use naming conventions for variables, constants, functions and classes
 
-**TL;DR:** Use **_lowerCamelCase_** when naming constants, variables and functions and **_UpperCamelCase_** (capital first letter as well) when naming classes. This will help you to easily distinguish between plain variables/functions, and classes that require instantiation. Use descriptive names, but try to keep them short
+**TL;DR:** Use **_lowerCamelCase_** when naming constants, variables and functions and **_UpperCamelCase_** (capital first letter as well) when naming classes, **_SCREAMING_CASE_** when naming global or static variables. This will help you to easily distinguish between plain variables/functions, classes that require instantiation and variables declared at global module scope. Use descriptive names, but try to keep them short
```

Comment:
I think we should add somewhere why screaming case 🤔

Reply:
The more conservative name is "upper snake case".

Reply:
> I think we should add somewhere why screaming case 🤔

I've changed convention name to the upper snake case. 

Regarding statement why, I think this statement explains it: 
`This will help you to easily distinguish ...  variables declared at global module scope`.
Or does it need separation from the rest?

Reply:
What I meant is maybe show example of this convention is used in Node (like PI as you mentioned in the issue) but on a second thought I think it should be here but rather in the code example